src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheckTest.java

```diff
@@ -73,7 +75,7 @@ public void testDefault() throws Exception {
             "136:12: " + getCheckMessage(MSG_KEY, "."),
             "264:2: " + getCheckMessage(MSG_KEY, "."),
         };
-        verify(checkConfig, getPath("InputWhitespace.java"), expected);
+        verify(checkConfig, getPath("InputNoWhitespaceAfterWhitespace.java"), expected);
```

Comment:
Name "NoWhitespaceAfterWhitespace" is confusing to me.
We don't have InputNoWhitespaceAfter.java in this folder yet. Please rename file.